pkg/watermark/processor/entity.go

```diff
@@ -21,6 +21,12 @@ func (w Watermark) String() string {
 	return t.Format(time.RFC3339)
 }
 
+func (w Watermark) Unix() int64 {
+	var location, _ = time.LoadLocation("UTC")
```

Comment:
should we move this as a global var? else we will have to keep loading all the time. perhaps do via `func init() {....}()` and `panic` if we cannot load the location

Reply:
Also, what will happen if we do not do LoadLocation? Are we seeing failures? All of our timestamps should be in UTC. If this is mostly for tests, then let's do LoadLocation only in tests.

Reply:
Agree. If we don't load location, the default location will be UTC and the behaviour will be the same(no failures). I will remove loading location here.